packages/protobuf-test/src/json-names.test.ts

```diff
@@ -13,115 +13,58 @@
 // limitations under the License.
 
 import { expect, test } from "@jest/globals";
-import { mkdtempSync, readFileSync, writeFileSync } from "fs";
-import { join } from "path";
-import { tmpdir } from "os";
-import { spawnSync } from "child_process";
 import { FileDescriptorSet, proto3, ScalarType } from "@bufbuild/protobuf";
+import { UpstreamProtobuf } from "upstream-protobuf";
```

Comment:
Since we're here, I've also added the `compileToDescriptorSet` method, which gives us the generic option to generate descriptors from inline protobuf source, and makes this test a bit more readable.